docs: change docs to add/edit/remove credentials (#985)
devopness/devopness: docs/credentials/get-credential-status.md

---
title: Get a Credential’s Status

> intro: Sometimes you may want to verify if a credential currently has the required permissions and access in a given provider. Get the Credential Status to validate if a credential is still active and has the necessary permissions.

Reviewer: Could you use the article name in the intro?

I believe that helps the user fixate the article's goal

```suggestion
intro: Sometimes you may want to verify if a credential currently has the required permissions and access in a given provider. Get a Credential's Status to validate if a credential is still active and has the necessary permissions.
```

> intro: Sometimes you may want to verify if a credential currently has the required permissions and access in a given provider. Get a Credential's Status to validate if a credential is still active and has the necessary permissions.

Reviewer 2: This is not what `credential:get_status` does.

```suggestion
intro: Get a Credential's Status to validate if a credential is still active and remains accepted by its provider.
```
